meetup/tests.py

```diff
@@ -113,95 +154,102 @@ def test_invalid_url_returns_404(self):
-    i_fetch = MeetupInfoFetch([self.meetup], mock_agent)
-    self.assertEquals('Mon May  4 08:00:00', i_fetch.fat_meetups()[0]['datetime'])
-
-  def test_fat_meetups_returns_map_link(self, mock_agent):
-    mock_agent.return_value.get_response.return_value.json.return_value = self.info
-    i_fetch = MeetupInfoFetch([self.meetup], mock_agent)
-    self.assertEquals("https://www.google.com/maps?q=40.7599983215332,-73.98999786376953", i_fetch.fat_meetups()[0]['map_link'])
+    self.info = {'venue'            : 'Some place',
+                 'next_event_topic' : 'awesome stuff',
+                 'datetime'         : 1426723243000,
+                 'lat'              : 40.75501251220703,
+                 'lon'              : -73.97337341308594}
+    self.meetup = Meetup(name="Meetup1", meetup_link='http://www.meetup.com/papers-we-love/')
+
+  def test_meetup_presenter_returns_meetup_link(self):
+    presenter = MeetupPresenter(self.meetup, self.info)
+    self.assertEquals(presenter.get_meetup_link(), 'http://www.meetup.com/papers-we-love/')
+
+  def test_meetup_presenter_returns_meetup_name(self):
+    presenter = MeetupPresenter(self.meetup, self.info)
+    self.assertEquals(presenter.get_meetup_name(), 'Meetup1')
+
+  def test_meetup_presenter_returns_meetup_venue(self):
+    presenter = MeetupPresenter(self.meetup, self.info)
+    self.assertEquals(presenter.get_meetup_venue(), 'Some place')
+
+  def test_meetup_presenter_returns_meetup_next_topic(self):
+    presenter = MeetupPresenter(self.meetup, self.info)
+    self.assertEquals(presenter.get_meetup_next_event_topic(), 'awesome stuff')
+
+  def test_meetup_presenter_returns_meetup_datetime(self):
+    presenter = MeetupPresenter(self.meetup, self.info)
+    self.assertEquals(presenter.get_meetup_datetime(), "Thu Jan 22 11:56:40")
+
+  def test_meetup_presenter_returns_meetup_map_link(self):
+    presenter = MeetupPresenter(self.meetup, self.info)
+    self.assertEquals(presenter.get_meetup_map_link(), "https://www.google.com/maps?q=40.75501251220703,-73.97337341308594")
+
+  @patch("meetup.services.meetup_api.MeetupApi")
+  def test_meetup_presenter_returns_pizza_places(self, fake_api):
+    self.meetup.save()
+    pizza_place = self.meetup.pizza_places.create(name="Pizza place")
+    presenter = MeetupPresenter(self.meetup, self.info)
+    self.assertEquals(presenter.get_meetup_pizza_places().first(), pizza_place)
+
+
+class TestMeetupApiResponseParser(TestCase):
+  def setUp(self):
+    self.mockResponse = HttpResponse()
+    self.mockResponse.json = MagicMock(return_value=meetup_api_response)
+
+  def test_parsed_response_contains_venue(self):
+    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('venue'), 'The Lexington')
+
+  def test_parsed_response_contains_event_topic(self):
+    self.assertEquals(meetup_api_response_parser.parse(self.mockResponse).get('next_event_topic'), 'Code & Coffee')
```

Comment:
I think you already configured this, but go ahead and go up to 120-160 characters.